static/js/util/test_utils.js

```diff
@@ -16,3 +18,13 @@ export function createAssertReducerResultState<State>(store: Store, getReducerSt
     }
   };
 }
+
+export const generateFakeKlasses = (numKlasses = 1) => {
+  return _.times(numKlasses, (i) => ({
+    klass_name: `Bootcamp 1 Klass ${i}`,
+    klass_key: i + 1,
+    payment_deadline: moment(),
+    total_paid: 0,
+    price: 1000
+  }));
+};
```

Comment:
might be worthwhile to put these sorts of functions in a `factories` subdirectory (`static/js/factories`) like we do in MM